ddl/table.go

```diff
@@ -51,7 +51,6 @@ func (d *ddl) onCreateTable(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 	switch tbInfo.State {
 	case model.StateNone:
 		// none -> public
-		job.SchemaState = model.StatePublic
```

Comment:
Why remove this?

Reply:
I don't remove it, it is handled in `Finish`.